cmake/CodeFormat.cmake

```diff
@@ -8,7 +8,7 @@ if(CLANG_FORMAT)
   file(GLOB_RECURSE ALL_SOURCE_FILES ${CLANG_FORMAT_CXX_FILE_EXTENSIONS})
 
   # Don't include some common build folders
-  set(CLANG_FORMAT_EXCLUDE_PATTERNS ${CLANG_FORMAT_EXCLUDE_PATTERNS} "/CMakeFiles/" "cmake" "thirdparties" "code_generator/kpsr_codegen/templates")
+  set(CLANG_FORMAT_EXCLUDE_PATTERNS ${CLANG_FORMAT_EXCLUDE_PATTERNS} "/CMakeFiles/" "cmake" "thirdparties" "code_generator/kpsr_codegen/templates" "code_generator/build" )
```

Comment:
also add "kidl" and "gen" --> kidl is the folder that holds custom message files (in kpe-admin) and "gen" is the folder of generated files (by code_generator). Normally these folder names are user defined, but we follow this naming for kpsr-tutorial, kpsr-robotics and kpe-admin, so we could add this here.

Reply:
Done